styles/pages/zoning/zoning.scss

```diff
@@ -7,18 +7,26 @@ $medium: 28px;
 
 .zoning {
 	width: 100%;
-	padding: 4px 4px;
+	padding: 8px;
 	flow-children: down;
 	overflow: squish;
 
+	&__title {
+		color: white;
+		font-family: $font-header;
+		font-size: 40px;
+	}
```

Comment:
Oh I noticed this the other day with `&__header` -- these don't have proper font styling.

Standard way is to import `@use ../../abstract/mixin` and use `@include mixin.font-styles($use-header: true);` in this block (and `&__header`), that'll gives you correct shadows and other properties.